packages/react-devtools-shell/webpack-server.js

```diff
@@ -25,12 +28,6 @@ if (!NODE_ENV) {
   process.exit(1);
 }
 
-const TARGET = process.env.TARGET;
```

Comment:
Why is this removed?

Reply:
Because its not needed, we are not building `react-devtools-shell` to static files, it is only used as a playground to test inline version of DevTools.

So the only target is `local` (to start dev-server)